### ISSUE TITLE
[Feature] Select events to trigger

### ISSUE THREAD

**Issue by Nisgrak** [abramenal/cypress-file-upload#120]
### Current behavior:
The actual lib call dragcenter, drop and dragleave events, some libs like Quasar needs others events.

In the case of Quasar I use the following code, modifing the lib and deleting the trigger to dragleave
```js
cy.get(".q-uploader__list").trigger("dragover");
cy.fixture("archivoAlumnos.xls").then(fileContent => {
	cy.get(".q-uploader__dnd").upload(
		{ fileContent, fileName: "archivoAlumnos.xls", mimeType: "application/vnd.ms-excel", encoding: "utf8" },
		{ subjectType: "drag-n-drop" },
	);
});
```

### Desired behavior:
The most ideal case should be the only use of fixture and upload command, without the previous trigger and deleting the dragleave event, a possibility to call user-defined events.

Thanks for the lib! 😄

**Comment by abramenal** [comment]
Hi @Nisgrak 
Thanks for submitting the issue!

I really expected this feature request. It looks logical and makes the plugin compliant to drag events DOM spec. However I was lazy enough not to include it at the beginning 😈 

So yes, let's do it!
Actually it might be a good open source contribution. All such people take their place in our honored contributors list in README.md.
Is that sound good to you? I expect only some changes here:
https://github.com/abramenal/cypress-file-upload/blob/master/src/handlers/handleDragDrop.js#L14

**Comment by Nisgrak** [comment]
Hey! I submit a PR a week ago, you have some time to look at it? Thanks :smile:

**Comment by abramenal** [comment]
Hey @Nisgrak, thanks for nudging me 😄 
It's merged, I've also applied some changes (638c92ebdaa7b1d736e72b00a410ae6bdba8ba2a).
Released in `v3.5.0`.
